packages/lit/rollup.config.js

```diff
@@ -58,11 +79,30 @@ export default litProdConfig({
       file: 'index',
       output: 'lit.min',
       format: 'es',
+      sourcemapPathTransform: (relativeSourcePath, sourcemapPath) => {
+        // Convert the paths of the sources to appear as descendants of a
+        // directory with the same name as the bundle. (By default, the paths in
+        // the source map are relative paths from the bundle to the included
+        // sources as they were on disk at the time the bundle was generated.
+        // This causes the developer tools to display the original sources at
+        // seemingly unrelated locations.)
+        return path.join(
+          'lit.min.js',
```

Comment:
Do we want the bundle name in there? If that's left off won't the path be relative to the folder containing the bundle, and be pretty good? Seems maybe weird to treat the file as a folder too. I wonder if that would trip up tools that try to display a file hierarchy?

Reply:
I tried loading it in Chrome and Firefox and they both seem to be fine with having a file and directory with the same name. IMO, it's clearer to keep them in this position rather than making them siblings because the folder is shown right next to the bundle vs. being potentially spread out across the parent folder.